helm/values-uat.yaml

```diff
@@ -1,25 +1,31 @@
 microservice-chart:
-  namespace: "your-namespace" # TODO: set your AKS namespace
+  namespace: "gps"
   nameOverride: ""
   fullnameOverride: ""
   image:
-    repository: pagopaucommonacr.azurecr.io/yourname # TODO
-    tag: "0.0.3"
+    repository: pagopaucommonacr.azurecr.io/pagopagpdpaymentsservice
+    tag: "0.0.1"
     pullPolicy: Always
   livenessProbe:
     httpGet:
-      path: /actuator/health/liveness
+      path: /info
       port: 8080
     initialDelaySeconds: 90
     failureThreshold: 6
     periodSeconds: 10
   readinessProbe:
     httpGet:
-      path: /actuator/health/readiness
+      path: /info
       port: 8080
     initialDelaySeconds: 90
     failureThreshold: 6
     periodSeconds: 10
+  startupProbe:
```

Comment:
i'm not sure this is necessary, see [here](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-readiness-probes)

Reply:
Our doubt is that if readiness and liveness execute simultaneously, delay may be missed and an early restart initiated

Reply:
Furthermore, probably, taking into account the failing startup scenario, the effect of startup probe is not comparable with the action triggered by readiness probe. The last one effect is the inbound traffic stop, while start probe, trigger  pod’s restartPolicy.

So our doubt is this: if we consider the scenario without startup probes and with a very long startup phase, readiness interrupts traffic and liveness restart the pod, seems that this behaviour does not solve the problem of the wait necessary for startup.

If the above scenario is yet covered by readiness and liveness probe, startup probe is useless.